src/cluster/setup.rst

```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
 This documentation will use the range TCP ``9100-9200``. Open up those ports in
 your firewalls and it is time to test it.
 
+Configure And Test The Communication With Erlang
+================================================
+
+Make CouchDB use correct IP|FQDN and the open ports.
+----------------------------------------------------
+
+In file etc/vm.args change the the line ``-name couchdb@127.0.0.1`` to
+``-name couchdb@<this-nodes-ip-address|FQDN>`` for each node which defines
+the node and must be seperate for each node. For clustered setup, each node in
```

Comment:
This sentence confuses me.  Do you mean "for each line which defines the node"?

And what does "must be separate for each node" mean?   A unique line per node?